test/dap-test.el

```diff
@@ -139,21 +139,34 @@
 /* \" // non-string */
 \"// string\"
 \"/*string*/\"")
-         (post-exp (dap-launch-test--delete-string-comments orig)))
+         (post-exp (dap-launch-test--sanitize-json orig)))
     (should (string= post-exp "
 
 
 
 \"// string\"
 \"/*string*/\""))))
 
+(ert-deftest dap-launch-test--delete-commas ()
+  (let ((orig "{
```

Comment:
```suggestion
  (let* ((orig "{
```
you're using earlier variables in let, meaning you must use let*.

Reply:
🤦